package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -209,3 +218,155 @@ def __init__(self, strand1: List[Residue], strand2: List[Residue],
             sel2.append(s[1].atoms.select_atoms(f'name {a2}'))
 
         super(WatsonCrickDist, self).__init__(sel1, sel2, **kwargs)
+
+
+class MinorPairDist(NucPairDist):
+    r"""Minor-Pair basepair distance for selected residues over a trajectory.
+
+    Takes two lists of :class:`~MDAnalysis.core.groups.Residue` objects and
```

Comment:
Can these be `ResidueGroup`, too? That would tie in even better with the OO approach.

Reply:
I'm looking at the source and docs for `ResidueGroup`. I can see the utility in merging `NucPairDist` with it, since it would allow for easier access to the residue related data in the classes. I'm a little unclear though on how it would handle the two strands the user provides, can `GroupBase` automatically deal with that if I add a line with `super(ResidueGroup, strand1, strand2)`.

Reply:
I didn't think that far ahead: All I thought was that instead of `strand1: List[Residue]` use `strand1: ResidueGroup` (etc).

Reply:
However, we then should also use ResidueGroup for WatsonCrick and then we're back to API changes... so if we use ResidueGroup here then we also need to deprecate `List[Residue]` for `WatsonCrick`.

I think I like `ResidueGroup` better because `strand = u.select_atoms("nucleic and resid 1-17").residues` looks better than `strand = list(u.select_atoms("nucleic and resid 1-17").residues)`. I _think_ one can simply convert a `res_list: List[Residue]` to `residues: [ResidueGroup]` with
```python
residues = ResidueGroup(res_list)
```
so accepting both for a transition period wouldn't be difficult.

Reply:
Ok, I'll make it clear in my code that either can be used for these processes.